stdlib/REPL/src/latex_symbols.jl

```diff
@@ -436,6 +436,7 @@ const latex_symbols = Dict(
     "\\Re" => "ℜ",
     "\\xrat" => "℞",
     "\\trademark" => "™",
+    "\\ohm" => "Ω",
```

Comment:
`Ω` here is U+03a9, but it should be U+2126?

Reply:
Oh interesting, I thought they normalized to the same utf-8 character in the file. How can you tell it is U+03a9?

Reply:
The normalization doesn't happen in the file, it happens only when Julia parses it.  To see what codepoint it was, I just copied and pasted it into the REPL from this PR:
```jl
julia> 'Ω'
'Ω': Unicode U+03a9 (category Lu: Letter, uppercase)
```

Reply:
Neat! I didn't know I could copy paste it and display it as a character in the Julia REPL.